javav2/example_code/entityresolution/src/main/resources/data.json

```diff
@@ -0,0 +1,3 @@
+{"id":"1","name":"Alice Johnson","email":"alice.johnson@example.com"}
```

Comment:
Make sure all of the fictitious names are from the safe name list. https://alpha.www.docs.aws.a2z.com/awsstyleguide/latest/styleguide/safenames.html#people

Reply:
fixed